script.js

```diff
@@ -1,12 +1,78 @@
 //You can edit ALL of the code here
```

Comment:
I like that you divided the code for the blocks that you described. It makes your code more readable.  👍